docs/source/IO.rst

```diff
@@ -0,0 +1,63 @@
+=========================
```

Comment:
nit: should we use io.rst as the file name to keep consistent with other files?

Reply:
Good point, will change.